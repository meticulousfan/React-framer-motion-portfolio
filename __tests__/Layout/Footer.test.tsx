/**
 * @jest-environment jsdom
 */

import { render } from "@testing-library/react";

import Footer from "../../src/components/Layout/Footer.component";

describe("Footer", () => {
  it("Footer laster inn og kan vises", () => {
    const { queryByTestId } = render(<Footer />);
    expect(queryByTestId("footer")).toBeTruthy();
  });
});
