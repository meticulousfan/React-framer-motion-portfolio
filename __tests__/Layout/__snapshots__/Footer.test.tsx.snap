// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Footer laster inn og kan vises 1`] = `
<div>
  <footer
    aria-label="Innholdet for bunnteksten med copyright"
    role="contentinfo"
  >
    <div
      class="mt-10 bg-gray-800 shadow"
    >
      <div
        aria-label="Copyright tekst"
        class="container p-6 mx-auto text-center text-white font-semibold inline-block"
      >
        Copyright Daniel Fjeldstad
        <svg
          class="inline-block m-2"
          fill="currentColor"
          height="1.2em"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 24 24"
          width="1.2em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M12 22c5.421 0 10-4.579 10-10S17.421 2 12 2 2 6.579 2 12s4.579 10 10 10zm0-18c4.337 0 8 3.663 8 8s-3.663 8-8 8-8-3.663-8-8 3.663-8 8-8z"
          />
          <path
            d="M12 17c.901 0 2.581-.168 3.707-1.292l-1.414-1.416C13.85 14.735 12.992 15 12 15c-1.626 0-3-1.374-3-3s1.374-3 3-3c.993 0 1.851.265 2.293.707l1.414-1.414C14.582 7.168 12.901 7 12 7c-2.757 0-5 2.243-5 5s2.243 5 5 5z"
          />
        </svg>
        2022
      </div>
    </div>
  </footer>
</div>
`;
